agent/src/agent/cli/run_test_pipeline.py

```diff
@@ -0,0 +1,102 @@
+import os
+import time
+import click
+import sdc_client
+from jsonschema import SchemaError, ValidationError
+
+from agent import destination, pipeline, source
+from agent.modules import constants
+from agent.modules.logger import get_logger
+
+logger = get_logger(__name__, stdout=True)
+
+
+@click.command(name='run-test-pipeline')
+def run_test_pipeline():
+    """
+    Test if everything works.
+    """
+    try:
+        _check_prerequisites()
+        _add_source()
+        _add_pipeline()
+        _run_pipeline()
+    except Exception as e:
+        click.secho(f'Test run failed: {e}', fg='red')
+    else:
+        click.secho('Test run OK', fg='green')
```

Comment:
if the pipeline got error responses from the destination it says `Test run OK` anyway
```
Pipeline run_test_pipeline info: status: RUNNING metrics: In: 20 - Out: 1 - Errors 20 (100.0%)
Test run OK
```

Also lets highlight all info messages, for example with yellow, because we show quite a lot of logs and it's hard to see important ones